internal/db/schema/migrations/postgres/2/84_oidc.up.sql

```diff
@@ -25,21 +26,26 @@ create table auth_oidc_method (
     constraint client_secret_hmac_not_empty
     check(length(trim(client_secret_hmac)) > 0),
   key_id wt_private_id not null -- key used to encrypt entries via wrapping wrapper. 
-    references kms_database_key_version(private_id) 
-    on delete restrict
-    on update cascade, 
+    constraint key_id_fkey
+      references kms_database_key_version(private_id) 
+      on delete restrict
+      on update cascade, 
   max_age int  -- the allowable elapsed time in secs since the last time the user was authenticated. A value -1 basically forces the IdP to re-authenticate the End-User.  Zero is not a valid value. 
     constraint max_age_not_equal_zero
       check(max_age != 0)
     constraint max_age_not_less_then_negative_one
       check(max_age >= -1), 
-  foreign key (scope_id, public_id)
-      references auth_method (scope_id, public_id)
-      on delete cascade
-      on update cascade,
-  unique(scope_id, name),
-  unique(scope_id, public_id),
-  unique(scope_id, discovery_url, client_id) -- a client_id must be unique for a provider within a scope.
+  constraint scope_id_public_id_fkey
+    foreign key (scope_id, public_id)
+        references auth_method (scope_id, public_id)
+        on delete cascade
+        on update cascade,
+  constraint scope_id_name_unique
+    unique(scope_id, name),
```

Comment:
For unique constraint names, I thought we were going with the naming convention of `{table_name}_col1_col2_uq` since unique constraint names must be unique within the schema?

Reply:
ty!